src/SourceBuild/content/eng/build.targets

```diff
@@ -0,0 +1,29 @@
+<Project>
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="FindArchiveDiffs" />
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="GetClosestOfficialSdk" />
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="GetSingleArchiveItem" />
+
+  <Target Name="ReportSdkArchiveDiffs"
+          AfterTargets="Build"
+          DependsOnTargets="DetermineSourceBuiltSdkVersion" >
+
+      <GetSingleArchiveItem SdkArchiveItems="@(SdkTarballItem)">
+        <Output TaskParameter="BestSdkArchiveItem" PropertyName="_BuiltSdkArchivePath"/>
+      </GetSingleArchiveItem>
+
+      <GetClosestOfficialSdk BuiltSdkPath="$(_BuiltSdkArchivePath)" Condition="'$(_BuiltSdkArchivePath)' != ''">
+        <Output TaskParameter="ClosestOfficialSdkPath" PropertyName="ClosestOfficialSdkPath" />
```

Comment:
Nit: You use the `_` prefix above for an internal property above, but don't here. I'd use the `_` in both cases.